packages/core/src/Modal/Modal.js

```diff
@@ -62,14 +62,15 @@ const centeredContent = resolve`
  */
 export const Modal = ({
     children,
-    onClose,
-    small,
-    large,
     className,
-    position,
     dataTest,
-}) => (
-    <Layer onClick={onClose} level={layers.blocking} translucent>
+    hide,
+    large,
+    onClose,
+    position,
+    small,
+}) => {
```

Comment:
This should be on L33 in this file, but the review wouldn't let me put it there so inlining it in the render function for illustration.

```suggestion
}) => {
    const layerStyles = resolve`
        div {
            background: none;
        }
    `
```